### PR TITLE
Update instructions for running local ETH2 Beacon/Execution chain pair

### PR DESCRIPTION
**Pull request by sourdzl**
tried to follow instructions, added a couple small clarifications

## 📝 Summary

some updates and missing steps for the setup instructions for a local PoS ETH2 net

## ⛱ Motivation and Context

helps with developing against a local ETH network

## 📚 References

---

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test`
* [x] `make run-mergemock-integration`
* [x] `go mod tidy`

### CODE REVIEW

**Comment by sourdzl** [comment]
fwiw, we are still unable to get the PoS chain running locally.  but this clarifies some of the instructions.

main error is the lighthouse beacon chain client not having an "execution layer" during the merge with `geth`.

File: docs/geth-pos-privnet.md [hunk 1]
```diff
@@ -10,45 +10,51 @@ Special thanks to Mateusz ([@mmrosum](https://twitter.com/mmrosum)) who put the
...
         2. Set `GENESIS_DELAY` to 30
         3. Set `ALTAIR_FORK_EPOCH` to 1
         4. Add `MERGE_FORK_EPOCH=1`
         5. Adjust `SECONDS_PER_SLOT, SECONDS_PER_ETH1_BLOCK, BN_COUNT`to your preference
             1. There seem to be some issues with multiple beacon nodes using the same EL, if doesn’t work set `BN_COUNT` to 1
         6. Do not change `VALIDATOR_COUNT, GENESIS_VALIDATOR_COUNT` to less than 64
+        7. modify VC_ARGS line to `VC_ARGS="--suggested-fee-recipient <address>"`, where <address> is the same public key that you registered in the `geth` command #8 above
     2. Modify scripts/local_testnet/beacon_node.sh:
-        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints [http://127.0.0.1:8545](http://127.0.0.1:8545/) --execution-endpoints [http://127.0.0.1:8545](http://127.0.0.1:8545/) --http-allow-sync-stalled`
+        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints http://127.0.0.1:8545/ --execution-endpoints http://127.0.0.1:8545/ --http-allow-sync-stalled`
```
Comment (michaelsproul) [inline docs/geth-pos-privnet.md:45]:
```suggestion
        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints http://127.0.0.1:8545/ --execution-endpoints http://127.0.0.1:8551/ --http-allow-sync-stalled`
```

I think you need to use port 8551 for Geth's engine API. The :8545 endpoint is unencrypted and AFAIK won't expose the engine API (certainly not by default).

Comment (michaelsproul) [reply docs/geth-pos-privnet.md:45]: You'll need to establish a JWT for the connection in this case. If you want to follow the path of least resistance I'd aim to minimise differences between Geth and Lighthouse's config here, and the config used on public merge testnets, e.g. Ropsten: https://github.com/remyroy/ethstaker/blob/main/merge-ropsten.md#configuring-your-geth-node

**Comment by metachris** [comment]
Great, thanks for this contribution! It's got a conflict now, and it would be great if you'd merge the discussions/suggestions too. Looking forward to merging this to improve the docs!

File: docs/geth-pos-privnet.md (hunk 1 above)
Comment (avalonche) [inline docs/geth-pos-privnet.md]:
```suggestion
        1. Add merge options: `--eth1 --merge --terminal-total-difficulty-override=60000000 --eth1-endpoints http://127.0.0.1:8545/ --execution-endpoints http://127.0.0.1:8551/ --http-allow-sync-stalled`
```

**Comment by metachris** [comment]
Thanks for putting this up, and @avalonche for moving this forward. 

For reference, there's this comment: https://github.com/flashbots/mev-boost/pull/187#discussion_r914413467

> You'll need to establish a JWT for the connection in this case. If you want to follow the path of least resistance I'd aim to minimise differences between Geth and Lighthouse's config here, and the config used on public merge testnets, e.g. Ropsten: https://github.com/remyroy/ethstaker/blob/main/merge-ropsten.md#configuring-your-geth-node

Should we work this in?

**Comment by michaelsproul** [comment]
At a glance it seems the current config is good, it's using a JWT and the authenticated API

**Comment by metachris** [comment]
thank you 👍

**Pull request merged**